content/workload-reduction-toolkit/evaluate-workload-measures/evaluate-progress-with-staff.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Evaluate progress with staff
+layout: /markdown_resource.html.erb
+colour: green
+---
+
+# Evaluate outcomes and share progress with staff
+
+<strong class="govuk-tag">Template</strong>
+
+Use this template to show your staff the actions you have taken, or will take, to address workload issues identified in
+your school. Gather feedback and evaluate if any actions taken have had an impact on their workload.
+
+<div class="govuk-grid-row dfe-width-container govuk-!-padding-bottom-6">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/download-icon.svg" alt="Clipboard icon">
+      </div>
+      <h2 class="govuk-heading-m">
+        Download the evaluating outcomes table
```

Comment:
```suggestion
        Download the template
```
The title is this in the changelog - not sure which one is correct! cc @ashleighk27